kratos/includes/slip_constraint.h

```diff
@@ -0,0 +1,352 @@
+//    |  /           |
```

Comment:
Constraints are supposed to be in a constraint folder, otherwise the includes folder is populated with many files (we should move linear_master_slave_constraint IMO, as it is not the base constraint)

Reply:
i am moving, but only the new file. 
moving the others should be done in a different PR, without any other modifications (other than what is needed)